src/lib.rs

```diff
@@ -27,6 +27,7 @@ pub static TOOLS: &[&str] = &[
     "rust-gdb",
     "rust-gdbgui",
     "rls",
+    "rust-analyzer",
```

Comment:
Should it be in DUP_TOOLS? That way it won't override a rust-analyzer someone directly installed in `.cargo/bin`.